### ISSUE TITLE
Node.js  Build WorkGroup Meeting 2024-04-23

### ISSUE THREAD

**Issue by mhdawson** [nodejs/build#3687]
## Time

**UTC Tue 23-Apr-2024 13:00 (01:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Tue 23-Apr-2024 06:00 (06:00 AM) |
| US / Mountain | Tue 23-Apr-2024 07:00 (07:00 AM) |
| US / Central  | Tue 23-Apr-2024 08:00 (08:00 AM) |
| US / Eastern  | Tue 23-Apr-2024 09:00 (09:00 AM) |
| EU / Western  | Tue 23-Apr-2024 14:00 (02:00 PM) |
| EU / Central  | Tue 23-Apr-2024 15:00 (03:00 PM) |
| EU / Eastern  | Tue 23-Apr-2024 16:00 (04:00 PM) |
| Moscow        | Tue 23-Apr-2024 16:00 (04:00 PM) |
| Chennai       | Tue 23-Apr-2024 18:30 (06:30 PM) |
| Hangzhou      | Tue 23-Apr-2024 21:00 (09:00 PM) |
| Tokyo         | Tue 23-Apr-2024 22:00 (10:00 PM) |
| Sydney        | Tue 23-Apr-2024 23:00 (11:00 PM) |

Or in your local time:
* https://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Build%20WorkGroup+Meeting+2024-04-23&iso=20240423T1300
* or https://www.wolframalpha.com/input/?i=01PM+UTC%2C+Apr+23%2C+2024+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1XZTAHqlpCdSd2PAaC_OIUIn62Q-Vp9Zr-72vcWkM27w/edit>

## Agenda

Extracted from **build-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

## Invited

* Build team: @nodejs/build

## Notes

The agenda comes from issues labelled with `build-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: <https://zoom.us/j/715960833>
* For those who just want to watch: We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at <https://www.youtube.com/c/nodejs+foundation/live> when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.
* youtube admin page: <https://www.youtube.com/my_live_events?filter=scheduled>

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure

**Comment by UlisesGascon** [comment]
I think that the agenda was not populated :eyes:

**Comment by richardlau** [comment]
> I think that the agenda was not populated 👀

@mhdawson could you take a look?

**Comment by mhdawson** [comment]
Created new issue that has the items. Not sure why it did not include them, but tweaked the template and the third issue now has them. Closing this issue in favor of the new one.
